sio-digital-team/active-safe
# Bundle all the js

[trello card](https://trello.com/c/HRTmWbES/49-bundle-all-the-js-1)


**Changes:**

- [x] Bundle js for dev
- [x] Bundle js for prod
- [x] Add es2015 support to gulp

Other 
- [x] added headings for prevent
- [x] removed unnecessary slider file
  
**How to test:**

- remove `dist` folder (if exists)
- run yarn watch
- in `dist/js` a file should appear called `bundle.js`
- site should look as usual
  

**Todo:**
